File: proep-mobile/app/src/main/java/com/proep/cliffin/models/Article.java
package com.proep.cliffin.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.Date;

public class Article {
    @SerializedName("newsId")
    @Expose
    int idNews;


    @SerializedName("title")
    @Expose
    String title;

    @SerializedName("body")
    @Expose
    String body;


    @SerializedName("timePublished")
    @Expose
    Date timePublished;

    @SerializedName("User_idUser")
    @Expose
    int userID;

    @SerializedName("image")
    @Expose
    private String image;

    public String getTitle() {
        return title;
    }

    public String getBody() {
        return body;
    }

    public Date getTimePublished() {
        return timePublished;
    }
}

File: proep-frontend/cliff-jumping-website/src/interfaces/Article.js
export class Article {

    id = null;
    title = "";
    body = "";
    timePublished = null;
    userId = null;
    image = "";
    constructor(id, title, body, timePublished, userdId, image) {
        this.id = id;
        this.title = title;
        this.body = body;
        this.timePublished = timePublished;
        this.userId = userdId;
        this.image = image;
    }
}